src/hooks/screen/useChatScreenHook.ts

```diff
@@ -99,6 +101,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           userId,
           selectedChannel?.id,
           message,
+          attachments,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` parameter is now passed to the `createChatSchema` function. This is a necessary change to include attachments in the chat schema.
```diff
+           attachments,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->